apps/avifenc.c

```diff
@@ -615,7 +615,7 @@ int main(int argc, char * argv[])
     uint32_t sourceDepth = 0;
     avifAppFileFormat inputFormat = avifInputReadImage(&input, image, &sourceDepth);
     if (inputFormat == AVIF_APP_FILE_FORMAT_UNKNOWN) {
-        fprintf(stderr, "Cannot determine input file format: %s\n", firstFile->filename);
+        fprintf(stderr, "Cannot determine input file format or read input file into avifImage: %s\n", firstFile->filename);
```

Comment:
Nit: I don't like `into avifImage` in the added text. Is `or read input file` sufficient here?

Reply:
Thanks for the review. The reason I wrote this pull request was that the avifImageRGBToYUV() call in avifJPEGRead() failed. So "read input file" won't describe that error. Maybe we can enumerate all possible errors?

`        fprintf(stderr, "Cannot determine input file format, read input file, or convert to YUV: %s\n", firstFile->filename);`